src/contents/members/kunalSinha.md

```diff
@@ -0,0 +1,5 @@
+---
+username:
+name: Kunal Sinha
+designation: Graphic Designer
```

Comment:
Nontech lead

Reply:
Should I then remove @priyanka2399  from the Non-Tech team ?